tests/Acceptance/ApplicationTest.php

```diff
@@ -119,7 +120,7 @@ public function test_chain_two_commands_with_a_comma_executes_both(): void
         $this->assertStringContainsString('All commands successfully executed!', MockWriter::$content);
         $this->assertStringNotContainsString('3 script(s) available', MockWriter::$content);
         $this->assertStringContainsString('Duration:', MockWriter::$content);
-        self::assertStringEqualsFile(__DIR__ . '/_app/result.txt', 'test');
+        $this->assertStringEqualsFile(__DIR__ . '/_app/result.txt', 'test');
```

Comment:
Previous implementation was correct.
```suggestion
        self::assertStringEqualsFile(__DIR__ . '/_app/result.txt', 'test');
```